internal/dataplane/parser/translate_failures_test.go

```diff
@@ -54,116 +52,6 @@ func TestTranslationFailureUnsupportedObjectsExpressionRoutes(t *testing.T) {
 				},
 			},
 		},
-		{
```

Comment:
Is this PR only include `TCPRoute` (suggest you to do so to avoid the PR being too large)? If so, this case should be kept but only remove the `TCPRoute` item.

Reply:
Sorry, I have restored the cases, because I thought the code is similar and simple, so I tried to support all L4 CRDs in this PR. So I open a PR for each CRD resource? @randmonkey

Reply:
IMO if you can keep the PR small enough (not more than 1000 lines of diffs -- ideally not more than 500 lines, and finish remaining things in 1~2 weeks), you can keep all CRDs inside one PR. Otherwise, it is better to split it into more smaller PRs, one for each CRD.